main.go

```diff
@@ -100,10 +104,10 @@ func run(flags userFlags) error {
 	}
 
 	// create the file
-	err = os.MkdirAll(filepath.Dir(flags.outFile), 0750)
+	err = os.MkdirAll(filepath.Dir(flags.outFile), 0o750)
```

Comment:
undo, `0o750` is in octal and equal to 488

Reply:
```suggestion
	err = os.MkdirAll(filepath.Dir(flags.outFile), 750)
```

Reply:
Thanks, one of the go tools that I use must have seen the preceeding 0's and changed it to octal automatically on format.

Reply:
Wait, the suggested change by @sudo-suhas is not correct. According to the Go spec, every integer that starts with a zero is octal, see https://go.dev/ref/spec#Integer_literals. The change to the `0o` prefix is a good thing, because it improves clarity. `0750` == `0o750`.

Reply:
Suppose I shouldn't have taken his feedback at face value. I will revert said change, thanks for clarifying @breml

Reply:
My apologies, I thought we were setting permissions to be `750`, didn't realise the number was _supposed_ to be in octal. Thanks for catching that @breml